nautobot/extras/models/models.py

```diff
@@ -131,7 +132,8 @@ def render_headers(self, context):
         if not self.additional_headers:
             return {}
         ret = {}
-        data = render_jinja2(self.additional_headers, context)
+        template_renderer = TemplateRenderer()
+        data = template_renderer.render_jinja2(self.additional_headers, context)
```

Comment:
Why this change?

Reply:
I believe the idea is that we want to be able to have the same rendering `Environment` wherever we render jinja templates so that custom jinja filters can be used anywhere. 
cc: @itdependsnetworks @jedelman8